pip/qsharp/interop/qiskit/backends/backend_base.py

```diff
@@ -277,7 +277,9 @@ def run_job(
         output["status"] = "COMPLETED"
         output["backend_name"] = self.name
         output["backend_version"] = self.backend_version
-        output["time_taken"] = str(time.time() - start)
+
+        duration = datetime.timedelta(seconds=(monotonic() - start))
```

Comment:
I assume you verified the resulting string is that same as was previously done on line 280 (or that it's not breaking if not)

Reply:
It was on my list to double check. You were right, the old value was float in secs. Could have sworn `time()` returned a struct for the `-` to give formatting.